auditbeat/module/file_integrity/eventreader_test.go

```diff
@@ -431,7 +435,8 @@ func rename(t *testing.T, oldPath, newPath string) {
 			return
 		}
 
-		if linkErr, ok := err.(*os.LinkError); ok && linkErr.Err == ErrorSharingViolation {
+		var e *os.LinkError
+		if errors.As(err, &e) && errors.Is(e.Err, ErrorSharingViolation) {
```

Comment:
```suggestion
		if errors.Is(err, ErrorSharingViolation) {
```
?
`(*os.LinkError).Unwrap` returns `e.Err`, so `errors.Is` will already be able to reach it.

Reply:
🤦🏻‍♂️ I think I fixed that in 2 steps and didn't notice